activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -100,7 +100,7 @@ def indexes(table_name)
       def index_exists?(table_name, column_name, options = {})
         column_names = Array(column_name).map(&:to_s)
         checks = []
-        checks << lambda { |i| i.columns == column_names }
+        checks << lambda { |i| Array(i.columns) == column_names }
```

Comment:
I think we should change `Index#columns` to always return an array.

Reply:
Done!

Reply:
Actually, this breaks the schema dumper, so I reverted this change. https://buildkite.com/rails/rails/builds/62458#d17be7e3-67df-49d9-89f3-87145b3bf30b